domains/justtakeabyte.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "JustTakeAByte",
+        "email": "trix7166@proton.me"
+    },
+    "record": {
+        "URL": "https://justtakeabyte.github.io/website/"
```

Comment:
website seems to be just blank, please add more stuff to the website

Reply:
Just added some more stuff